feat(ssl): add function to get request ssl pointer (#95)
Kong/lua-kong-nginx-module: README.md

[Back to TOC](#table-of-contents)

resty.kong.tls.get\_request\_ssl\_pointer
----------------------------------------------------
**syntax:** *ssl_ptr, err = resty.kong.get\_request\_ssl\_pointer()*

> **context:** *client_hello_by_lua&#42;, *ssl_certificate_by_lua&#42;, *rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, log_by_lua&#42;*, *preread_by_lua&#42;*

Reviewer: ```suggestion
**context:** *client_hello_by_lua&#42;, *ssl_certificate_by_lua&#42;, *rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, log_by_lua&#42;*
```

> **subsystems:** *http* *stream*

Reviewer: ```suggestion
**subsystems:** *http*
```

> Retrieves the OpenSSL `SSL*` object for the current tcpsock `sock`.

Reviewer: ```suggestion
Retrieves the OpenSSL `SSL*` object for the current HTTP request.
```
